src/libraries/System.Net.Quic/src/System/Net/Quic/QuicOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Net.Quic
+{
+    /// <summary>
+    /// Options for QUIC
+    /// </summary>
+    public class QuicOptions
+    {
+        /// <summary>
+        /// Limit on the number of bidirectional streams the remote peer connection can create on an open connection.
+        /// Default is 100.
+        /// </summary>
+        // TODO consider constraining these limits to 0 to whatever the max of the QUIC library we are using.
+        public long MaxBidirectionalStreams { get; set; } = 100;
```

Comment:
Need validation to stop invalid values (e.g. 0 or -50). Applies to all options.

Create an issue if you would prefer to do it in a follow up PR.

Reply:
It's part of this https://github.com/dotnet/runtime/issues/32079:
> How should we be validating these limits?